jujugui/static/gui/src/app/components/inspector/inspector.js

```diff
@@ -21,6 +21,9 @@ with this program.  If not, see <http://www.gnu.org/licenses/>.
 YUI.add('inspector-component', function() {
 
   juju.components.Inspector = React.createClass({
+    propTypes: {
```

Comment:
Didn't want to get sucked into adding ALL the propTypes for the `Inspector`. That can wait for another day and a dedicated card.